samples/customABSample/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
It looks like a default file. Can we remove it.

Reply:
Updated.